Gds/src/fprime_gds/common/loaders/xml_loader.py

```diff
@@ -114,11 +115,21 @@ def get_xml_tree(path):
         xml_parser = etree.XMLParser(remove_comments=True)
 
         with open(path) as fd:
-
             # Parse xml and get element tree object we can retrieve data from
             element_tree = etree.parse(fd, parser=xml_parser)
-
-        return element_tree.getroot()
+        root = element_tree.getroot()
+
+        # Check version of the XML before continuing. Versions weren't published before 1.5.4.  Only check major minor
+        # and point versions to allow for development versions to be allowed.
+        dict_version_string = root.attrib.get("framework_version", "1.5.4")
```

Comment:
Is it fine to always assume version 1.5.4 for all dictionaries without framework_version?

Reply:
Yes, as the next version will include a full versioning, and previous versions are all run by the GDS as 1.5.4 anyway.